config.json

```diff
@@ -311,6 +311,14 @@
         "prerequisites": [],
         "difficulty": 5
       },
+      {
+        "slug": "error-handling",
+        "name": "Error Handling",
+        "uuid": "a804dd5f-02cc-42e1-ab43-1f2c9ef0d0c4",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
Could you update the difficulty/ordering?

Reply:
Oh good catch, I keep forgetting, don't I!?